src/zope/testbrowser/browser.py

```diff
@@ -528,6 +528,12 @@ def toStr(self, s):
             return s
         if s is None:
             return None
+        # Might be an iterable, especially the 'class' attribute.
+        if isinstance(s, (list, tuple)):
+            subs = [self.toStr(sub) for sub in s]
+            if isinstance(subs, tuple):
```

Comment:
Ah, yes, my mistake. Fixed and squashed.
We might never encounter a tuple, but it is probably good to check for it anyway.